features/mirror_spec.rb

```diff
@@ -1,7 +1,8 @@
 require File.expand_path('support/command_rspec_helper', __dir__)
 
 describe 'mirror' do
-  let(:path) { '/var/lib/rmt/public/repo/SUSE/Products/SLE-Product-SLES/15-SP5/x86_64' }
+  let(:files) { %w[repomd.xml repomd.xml.asc repomd.xml.key *-primary.xml.gz *-filelists.xml.gz *-other.xml.gz] }
+  let(:path) { '/var/lib/rmt/public/repo/SUSE/Updates/SLE-Product-SLES/15' }
```

Comment:
```suggestion
  let(:path) { '/var/lib/rmt/public/repo/SUSE/Products/SLE-Product-SLES/15-SP5/x86_64' }  
```
the new path seems to be incorrect here. The old path works fine and fixes the broken tests.